### ISSUE TITLE
Character unable to move between rooms

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#295]
![image](https://user-images.githubusercontent.com/52435234/110050987-09ff8480-7d23-11eb-8837-9a24e11c773c.png)

the command sends, and the response comes through, character fails to move

**Comment by thaumatology** [comment]
fixed seemngly
